assets/javascripts/discourse/components/composer-presence-display.js.es6

```diff
@@ -0,0 +1,21 @@
+import computed from 'ember-addons/ember-computed-decorators';
+
+export default Ember.Component.extend({
+  composer: Ember.inject.controller(),
+
+  @computed('composer.presenceUsers')
+  users(){
```

Comment:
When you use the `@computed` decorator, you can (should) pass the observed values as arguments. So this would become

```javascript
@computed('composer.presenceUsers')
users(presenceUsers) {
  return presenceUsers.filter(user => user.id != this.get("currentUser").id);
}
```